src/Jackett.Common/Definitions/thepiratebay.yml

```
@@ -148,6 +148,8 @@ search:
     # this filter searches both formats, so "General Hospital S01E02" becomes "General Hospital S01E02 | (S01 E02)"
     - name: re_replace
       args: ["General Hospital S(\\d{2,3})E(\\d{2,3})", "$0 | \\(S$1 E$2\\)"]
+    - name: re_replace
+      args: ["\\bS0?(\\d{1,2})\\b", "$0 | \\(Season $1\\)"]
```

```suggestion
      args: ["\\bDoctor Who (?:2005 )?S0?(\\d{1,2})\\b", "$0 | \\(Season $1\\)"]
```

Make it specific to Doctor Who, like with the General Hospital re_replace?